src/io.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const { emitKeys, CSI, cursorTo } = require('./tty');
+const multiline = Symbol('IO.kNeedsAnotherLine');
```

Comment:
please name the variable `kNeedsAnotherLine`, and if it is supposed to be `IO.kNeedsAnotherLine`, put it on the `IO` object. (`IO.kNeedsAnotherLine = Symbol('IO.kNeedsAnotherLine')`